package.json

```
@@ -377,7 +377,7 @@
     "@types/eth-url-parser": "^1.0.0",
     "@types/i18n-js": "^3.8.4",
     "@types/is-url": "^1.2.30",
-    "@types/jest": "^28.1.8",
+    "@types/jest": "^29.5.4",
```

```suggestion
    "@types/jest": "^29.5.12",
```

but prob don't want to apply this suggestion, as it will miss updating yarn.lock

or can just bump this later

---

nbd!